runner/steps.ts

```diff
@@ -127,8 +106,7 @@ export function cancelTests(context: Context) {
 
 // Helpers
 
-function runBrowsers(
-      context: Context, done: (err?: any) => void): BrowserRunner[] {
+function runBrowsers(context: Context) {
```

Comment:
It became more complicated because completion is now separate, so I'm letting type inference handle things here.